infrastructure/variables.tf

```diff
@@ -58,3 +58,28 @@ variable "use_uk_db" {
 variable "idam_api_url" {
   default = "http://betaDevBccidamAppLB.reform.hmcts.net"
 }
+
+////////////////////////////////
+// Database
+////////////////////////////////
+
+variable "postgresql_user" {
+  default = "ccd"
+}
+
+variable "database_name" {
+  default = "ccd_definition"
+}
+
+variable "use_uk_db" {
+  type = "string"
+  default = "false"
+}
+
+////////////////////////////////
+// S2S
+////////////////////////////////
+
+variable "s2s_url" {
+  default = "http://betaDevBccidamS2SLB.reform.hmcts.net"
+}
```

Comment:
Please revert all of the changes for that file as they are duplicating existing work.